docs/.gitignore

```diff
@@ -0,0 +1,2 @@
+_build
```

Comment:
Hm...`.gitignore` here? Does it works? And isn't root `.gitignore` satisfiable?

Reply:
Yep, it works. I don't remember if I added that file myself or it was generated by `sphinx-quickstart` automatically; but I can move this stuff to the root `.gitignore` if you prefer.

Reply:
Hm...Let's keep gitignore within a single place. It feels like a flashback to `.htaccess` when you WTF about what's going on untill walk though whole FS tree to figure out why. IIRC, there should be some section about sphinx already, but it's easy to add a any one.